app/src/main/java/org/oppia/app/player/exploration/ExplorationActivity.kt

```diff
@@ -48,18 +50,22 @@ class ExplorationActivity : InjectableAppCompatActivity(), StopExplorationInterf
     internal const val EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY = "ExplorationActivity.topic_id"
     internal const val EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY = "ExplorationActivity.story_id"
     internal const val EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY = "ExplorationActivity.exploration_id"
+    internal const val EXPLORATION_ACTIVITY_BACKFLOW_SCREEN = "ExplorationActivity.backflow_screen"
```

Comment:
Rename this to `EXPLORATION_ACTIVITY_BACKFLOW_SCREEN_KEY`

Reply:
Done